src/telegraf.ts

```diff
@@ -183,20 +189,25 @@ class Telegraf<TContext extends Context = Context> extends Composer<TContext> {
   launch(config: Telegraf.LaunchOptions = {}) {
     debug('Connecting to Telegram')
     return this.telegram.getMe().then((botInfo) => {
-      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-      debug(`Launching @${botInfo.username}`)
+      debug(
+        `Launching ${botInfo.username == null ? 'bot' : '@' + botInfo.username}`
```

Comment:
Dead code, bots always have a username as far as I'm aware. The real solution here is to tighten the type returned by `getMe` and stored in `botInfo` to something like
```ts
interface BotUser {
  id: number
  is_bot: true
  first_name: string
  username: string
  can_join_groups: boolean
  can_read_all_group_messages: boolean
  supports_inline_queries: boolean
}
```

Reply:
Sure, we'll fix that later, I'd enjoy seeing some extending here, since BotUser would just be an extension of User in this case